### PR TITLE
Removed unnecessary aspects of StatMech

### PR DESCRIPTION
**Pull request by ndharms**
After chatting with @davidfarinajr, it was brought to our attention that we were over-specifying information in our Arkane input files. Specifically:

- We didn't need to specify the number of atoms and the atom types
- We didn't need to specify the number of bonds and the bond types
- We didn't need to specify the external symmetry

Arkane will do all of this on its own. This PR removes those specifications in `statmech.py` and corresponding tests. My one question: would there be a time when we would want to specify these information instead of relying on parameters determined from Arkane?

### CODE REVIEW

File: autotst/calculator/statmech.py [hunk 1]
```diff
@@ -224,30 +130,11 @@ def write_conformer_file(self, conformer):
...
+                  '# -*- coding: utf-8 -*-', ]
 
         output += ["",
                    "linear = {}".format(conformer.rmg_molecule.is_linear()),
                    "",
-                   "externalSymmetry = {}".format(external_symmetry),
-                   "",
                    "spinMultiplicity = {}".format(conformer.rmg_molecule.multiplicity),
                    "",
                    "opticalIsomers = 1",
```
Comment (davidfarinajr) [inline autotst/calculator/statmech.py]: `linear` and `opticalIsomers` are now optional. Arkane using 3D coords to determine if it's linear (which I think should be more accurate than the `molecule.is_linear()` method which uses 2D graph).  We should probably remove `linear` and `opticalIsomers` as well as the externalSym.

File: autotst/calculator/statmech.py [hunk 2]
```diff
@@ -375,32 +262,13 @@ def write_ts_input(self, transitionstate):
...
-
-            output.append("}")
-        else:
-            output.append('bonds = {}')
         transitionstate.rmg_molecule.update_multiplicity()
 
-        external_symmetry = transitionstate.calculate_symmetry_number()
-
         output += ["",
                    "linear = False",
```
Comment (davidfarinajr) [inline autotst/calculator/statmech.py]: I think we should remove `linear` and `opticalIsomers` here too

**Review by davidfarinajr — CHANGES_REQUESTED** [review]
Looks good, I think we should remove `linear` and `opticalIsomers` as well as they are now optional

File: autotst/calculator/statmech.py (hunk 2 above)
Comment (ndharms) [reply autotst/calculator/statmech.py]: Removed those sections, let's see how the tests go.

**Review by davidfarinajr — APPROVED** [review]

**Comment by ndharms** [comment]
Sorry, couldn't merge this at the moment, rebased and force pushed so once the tests pass, I'll merge in. Thanks for the review @davidfarinajr

**Pull request merged**